Dockerfile

```diff
@@ -1,6 +1,10 @@
-FROM ghcr.io/open-feature/flagd:v0.5.2
+FROM ghcr.io/open-feature/flagd:v0.6.3 as flagd
 
-COPY testing-flags.json testing-flags.json
+FROM busybox:1.36
```

Comment:
busybox because we need some basic tools and a shell.

Reply:
This is fine. Final image is ~60MB